### PR TITLE
squishyball (new formula)

### PR DESCRIPTION
**Pull request by tdsmith**
Submitted by an anonymous IRC user. Would need a FLAC fixture and
terminal interaction to do a better test.

### CODE REVIEW

File: squishyball.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Squishyball < Formula
+  homepage "http://xiph.org/"
```
Comment (DomT4) [inline squishyball.rb]: Both these links can roll SSL/TLS :).

**Comment by tdsmith** [comment]
I get certificate errors from svn if I try and use https://svn.xiph.org.

**Comment by DomT4** [comment]
Ah rats, yeah, the system ` svn ` doesn't work with the cert. That's a pain. Never mind on that then I guess :smile_cat:

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
:ship:'d it.

**Pull request closed without merging**